### ISSUE TITLE
Styling placeholders (potentially others)

### ISSUE THREAD

**Issue by mxstbr** [styled-components/styled-components#13]
This currently does not work at all, because our "parser" doesn't support it:

```CSS
::-webkit-input-placeholder {
  some: styles;
}
```

Possibly related to #6 if we end up implementing the PostCSS parser

**Comment by geelen** [comment]
Closed by using a proper parser in #29

**Comment by jorilallo** [comment]
@geelen @mxstbr how is this supposed to work? I tried with following and it has no effect:

```javascript
const EditArea = s.textarea`
  outline: none;

  ::-webkit-input-placeholder {
    color: red;
  }
`;
```
